Recon/src/Recon/RecoilFiducialityProcessor.cxx

```diff
@@ -0,0 +1,132 @@
+/**
+ * @file RecoilFiducialityProcessor.cxx
```

Comment:
if you touch this file again, this still has `camelCase` in it, altho they are local variables, so less problematic to rename them later